### DISCUSSION TITLE
Restoring PeraConnect session

### DISCUSSION THREAD

**Discussion by No-Cash-7970** [reach-sh/reach-lang#1349] [Q&A] [answered]
How do you restore a PeraConnect session from localStorage?

When using PeraConnect, it saves the session in the `PeraWallet.Wallet`, `PeraWallet.BridgeURL` and `walletconnect` keys in localStorage. The `restoreWalletConnect()` function in the algo-wallet-demo example doesn't seem to work for PeraConnect.

**Comment by jeapostrophe** [comment, marked as answer]
The example program doesn't demonstrate saving the PeraConnect session, but the exact technique would work. You create a `PeraWalletConnect` object yourself and pass it as an argument to the wallet fallback. You can see how it works right here: https://github.com/reach-sh/reach-lang/blob/master/js/stdlib/ts/ALGO_PeraConnect.ts#L8-L14
